src/architecture/msgPayloadDefC/CameraRenderingMsgPayload.h

```diff
@@ -0,0 +1,36 @@
+/*
+ ISC License
+
+ Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
:small_red_triangle_down: This copyright statement is incorrect. It should read `2024, Laboratory for Atmospheric Space Physics, University of Colorado at Boulder`. This is the same for the other message added in this PR.

Reply:
done